cime_config/allactive/config_pesall.xml

```diff
@@ -7087,6 +7050,66 @@
           <rootpe_cpl>0</rootpe_cpl>
         </rootpe>
       </pes>
+      <pes compset=".*EAM.+ELM.+MPASSI.+MPASO.+SGLC_SWAV_SIAC_SESP_BGC.*" pesize="M">
+        <comment>anvil: --compset BGC* --res ne30pg2_r05_EC30to60E2r2 on 30 nodes pure-MPI, ~3 sypd </comment>
```

Comment:
@amametjanov - the PR says it's setting up a small layout but it's specified M here?

Reply:
It's small in the sense of smallest Anvil node-count that can run BGC (for test defaults), not in the sense of `pesize="S"` small. I'll change to "default".

Reply:
that sounds great